src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -19,15 +27,71 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
-            throw new ParseException(
-                    String.format(MESSAGE_NO_PARAMETER_FOUND, FindCommand.MESSAGE_USAGE));
+        requireNonNull(args);
+
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_TELEGRAM_HANDLE, PREFIX_EMAIL,
+                        PREFIX_STUDENT_STATUS, PREFIX_ROLE, PREFIX_NICKNAME);
+
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_TELEGRAM_HANDLE,
+                PREFIX_EMAIL, PREFIX_STUDENT_STATUS, PREFIX_NICKNAME);
+
+        if (args.trim().isEmpty() || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
```

Comment:
I think it would be preferably if there was still a line that retained. "Please enter something for me to search". Distinguishes invalid input versus empty input